apps/asap-server/src/controllers/research-outputs.ts

```diff
@@ -120,6 +135,48 @@ export default class ResearchOutputs implements ResearchOutputController {
       ),
     };
   }
+  async create(
+    researchOutputData: ResearchOutput,
```

Comment:
Do you know what - i think we should separate these input types from the models, what are models anyway? squidex data models right? and here we want to build something detached from the implementation. I don't mind extending `ResearchOutput` type but i think we could define the controller input type down below and have the `teamId` be part of the payload?